mu-plugins/rest-api/endpoints/class-wporg-base-locale-banner-controller.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace WordPressdotorg\MU_Plugins\REST_API;
+
+/**
+ * Base_Locale_Banner_Controller
+ */
+class Base_Locale_Banner_Controller extends \WP_REST_Controller {
+	/**
+	 * Register the endpoint routes used across both themes and plugins.
+	 *
+	 * @see register_rest_route()
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				'methods' => \WP_REST_Server::READABLE,
+				'callback' => array( $this, 'get_response' ),
+				'args' => array(
+					'debug' => array(
+						'sanitize_callback' => function( $param ) {
+							return (bool) $param;
+						},
+					),
+				),
+				'permission_callback' => '__return_true',
+			)
+		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<slug>[^/]+)/',
+			array(
+				'methods' => \WP_REST_Server::READABLE,
+				'callback' => array( $this, 'get_response_for_item' ),
+				'args' => array(
+					'debug' => array(
+						'sanitize_callback' => function( $param ) {
+							return (bool) $param;
+						},
+					),
```

Comment:
This can just use the inbuilt boolean sanitization I think? 
```suggestion
					'debug' => array(
						'type' => 'boolean'
					),
```

Reply:
Oh yeah, I forgot there's built-in checking for basic types. Done in https://github.com/WordPress/wporg-mu-plugins/pull/609/commits/05c7766df287a3018aa5139b8519b7517034e675.